### PR TITLE
add service type metadata to kubernetes_sd_config service role #6496

### PR DESCRIPTION
**Pull request by liguozhong**
https://github.com/prometheus/prometheus/issues/6496

### CODE REVIEW

**Comment by simonpasquier** [comment]
you would need to update https://github.com/prometheus/prometheus/blob/master/docs/configuration/configuration.md too.

**Comment by liguozhong** [comment]
> you would need to update https://github.com/prometheus/prometheus/blob/master/docs/configuration/configuration.md too.

thanks , done

File: docs/configuration/configuration.md [hunk 1]
```diff
@@ -738,6 +738,7 @@ Available meta labels:
 * `__meta_kubernetes_service_annotationpresent_<annotationname>`: "true" for each annotation of the service object.
 * `__meta_kubernetes_service_cluster_ip`: The cluster IP address of the service. (Does not apply to services of type ExternalName)
 * `__meta_kubernetes_service_external_name`: The DNS name of the service. (Applies to services of type ExternalName)
+* `__meta_kubernetes_service_type`: The type of the service.
```
Comment (simonpasquier) [inline docs/configuration/configuration.md]: Please keep the list of labels sorted.

Comment (liguozhong) [reply docs/configuration/configuration.md]: done

**Comment by simonpasquier** [comment]
:+1:

**Comment by liguozhong** [comment]
> 👍

Thank you very much for your review and suggestions

**Comment by liguozhong** [comment]
> 👍

Will this PR merge to the master? What else needs to be done

**Pull request merged**